core/templates/dev/head/dashboard/Dashboard.js

```diff
@@ -34,10 +35,10 @@ oppia.controller('Dashboard', [
     };
 
     $rootScope.loadingMessage = 'Loading';
-    $http.get('/dashboardhandler/data').then(function(response) {
-      var data = response.data;
-      $scope.explorationsList = data.explorations_list;
-      $scope.collectionsList = data.collections_list;
+    DashboardBackendApiService.fetchDashboardData().then(function(response) {
```

Comment:
Can we display an error message/warning of some type if the promise is rejected?